contrib/syslog-ng-debun

```diff
@@ -1664,6 +1674,7 @@ setup_env
 debun_run
 [ "$syslogfhs" = "linux" ] && fhs_set_linux
 [ "$syslogfhs" = "unix" ] && fhs_set_unix
+save_config_file
```

Comment:
Does it align with the `$privacy_mode`  mode ?

Reply:
I think it will save config file in privacy mode, too.